.github/workflows/CI.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v3
       with:
         distribution: goreleaser
-        version: latest
+        version: "~> v1"
```

Comment:
What does this do?

Reply:
Since the last change goreleaser pushed a v2 that latest started pulling (and failing the CI due to a command removal). I felt updating to v2 of goreleaser was out of scope for this PR so I changed it to pull the most recent version of v1.